### ISSUE TITLE
Allow events to belong to more than one category

### ISSUE THREAD

**Issue by pferreir** [indico/indico#2178]
This is not by any means simple, but it's something we should consider at some point. The main issue seems to be protection.

**Comment by ThiefMaster** [comment]
One actual parent category, and just "links" in the other ones? I think anything else would be a huge mess in terms of protection (but also e.g. breadcrumbs).

**Comment by pferreir** [comment]
Yep, indeed. Or categories as tags, but that's something for later.
